docs/DeveloperGuide.md

```diff
@@ -1,38 +1,146 @@
 # Developer Guide
 
+* [Acknowledgements](#acknowledgements)
```

Comment:
I like the easy access to the different sections of the guide from the table of contents!